docs/arts-and-crafts.md

```diff
@@ -34,23 +34,10 @@ into real world art! Please drop us a message if you know another tool to be add
 ]
 ```
 
-## Toys and Pets
-
-```codecard
-[
-    {
-        "name": "KittyWumpus",
-        "description": "An adventure companion that joins with you in your game.",
-        "url": "https://imakeathome.us/kittywumpus",
-        "imageUrl": "/static/arts-and-crafts/kitty-wumpus.jpg"
-    }
-]
-```
-
 ## See Also
 
 * [Tshirts](/arts-and-crafts/tshirt.md)
 * [LEGO Bricks Murals](/arts-and-crafts/lego-bricks-murals)
 * [Laser-cut Stencils](/arts-and-crafts/laser-cut-stencils)
+* [Perler Bead Sprites](/arts-and-crafts/perler-bead-sprites)
 * [Arcade Stencilr](https://arcade-stencils.glitch.me/)
```

Comment:
Of course, any external links don't work either. We don't just remove all of our external links? I think the codecard logic could check the online status and "grey out" the linkout cards.